mage/template.go

```diff
@@ -31,6 +31,8 @@ func main() {
 	}
 
 	targets := map[string]bool {
+		{{range $alias, $funci := .Aliases}}"{{$alias}}": true,
```

Comment:
you should lowercase the alias, because we always lowercase the input, so an alias of "Foo" will never match.